chipsec/hal/interrupts.py

```diff
@@ -79,16 +70,11 @@ def send_SMI_APMC( self, SMI_code_port_value, SMI_data_port_value ):
         return self.cs.io.write_port_word( SMI_APMC_PORT, SMI_code_data )
 
 
-    def get_PMBASE(self):
-        return (self.cs.pci.read_dword( 0, 31, 0, Cfg.CFG_REG_PCH_LPC_PMBASE ) & ~0x1)
-
-    def get_TCOBASE(self):
-        return (self.get_PMBASE() + Cfg.TCOBASE_ABASE_OFFSET)
-
 
     def send_NMI( self ):
         if logger().HAL: logger().log( "[intr] sending NMI# through TCO1_CTL[NMI_NOW]" )
-        tcobase = self.get_TCOBASE()
+        reg, ba = self.cs.get_IO_space("TCOBASE")
+        tcobase = self.cs.read_register_field(reg,ba)
```

Comment:
NIT: not space after comma, the rest of the patch has a mixed style of space and not-space after comma.